dll/shellext/netshell/lang/zh-CN.rc

```diff
@@ -144,9 +144,9 @@ BEGIN
     IDS_SHV_COLUMN_TYPE "类型"
     IDS_SHV_COLUMN_STATE "状态"
     IDS_SHV_COLUMN_DEVNAME "设备名称"
-    IDS_SHV_COLUMN_PHONE "电话 # 或主机地址"
+    IDS_SHV_COLUMN_PHONE "电话号码或主机地址"
     IDS_SHV_COLUMN_OWNER "所有者"
-    IDS_TYPE_ETHERNET "局域网或高速互联网"
```

Comment:
revert it because it's better.

Reply:
In Windows (from NT5.2 to the latest Win11), "LAN" and "Internet" are preserved as well-known vocabulary, not translated.

![20240531011347](https://github.com/reactos/reactos/assets/55521781/e52dc90f-fcda-4593-8d92-62e9dd2f41a6)
![20240531011510](https://github.com/reactos/reactos/assets/55521781/85363fc3-1a6b-41c9-b67e-4608b50dfcef)
![20240531012008](https://github.com/reactos/reactos/assets/55521781/ad3fa631-3507-4134-9061-742d1418fe30)

Reply:
oh